### DISCUSSION TITLE
Custom Overlay

### DISCUSSION THREAD

**Discussion by centratelemedia** [inocan-group/vue3-google-map#233] [Q&A]
howto add custom overlay like Openstreetmap, Bing Map?

**Comment by HusamElbashir** [comment]
Hey

Don't know if you've seen the google official docs on [custom overlays](https://developers.google.com/maps/documentation/javascript/customoverlays). Refer to the [advanced usage](https://github.com/inocan-group/vue3-google-map#advanced-usage) section in the readme to learn how you can attach your overlay view to the map.
